src/definitions/behaviors/form.js

```diff
@@ -1207,13 +1207,7 @@ $.fn.form = function(parameters) {
               module.debug('Using field name as identifier', identifier);
               field.identifier = identifier;
             }
-            var isDisabled = true;
-            $.each($field, function(){
-                if(!$(this).prop('disabled')) {
-                  isDisabled = false;
-                  return false;
-                }
-            });
+            var isDisabled = !$field.filter(':not(:disabled):first').length;
```

Comment:
The `:first` selector does not seem to be necessary . It is also [deprecated as of Jquery 3.4](https://api.jquery.com/first-selector/).

Reply:
ok i will remove it. thought it's faster